spaceship/lib/spaceship/portal/portal_client.rb

```diff
@@ -363,16 +363,15 @@ def devices_by_class(device_class, include_disabled: false)
     def create_device!(device_name, device_id, mac: false)
       ensure_csrf(Spaceship::Device)
 
-      req = request(:post) do |r|
-        r.url "https://developerservices2.apple.com/services/#{PROTOCOL_VERSION}/#{platform_slug(mac)}/addDevice.action"
-        r.params = {
-          teamId: team_id,
-          deviceNumber: device_id,
-          name: device_name
-        }
-      end
+      req = request(:post, "account/#{platform_slug(mac)}/device/addDevices.action", {
```

Comment:
It's a bit weird that the `platform_slug` is still part of the  url now that `devicesClasses` is a parameter.